src/oaklib/implementations/sqldb/sql_implementation.py

```diff
@@ -1062,6 +1062,13 @@ def _rbox_relationships(
         for row in q:
             yield row.subject, row.predicate, row.object
 
+    def relationships_metadata(
+        self, relationships: Iterable[RELATIONSHIP], **kwargs
+    ) -> Iterator[Tuple[RELATIONSHIP, List[Tuple[PRED_CURIE, Any]]]]:
```

Comment:
I think this should be:
`Iterator[Tuple[RELATIONSHIP, List[Tuple[om.Annotation]]]]`

i.e. replace `PRED_CURIE, Any` with `om.Annotation`